app/renderer/actions/Session.js

```diff
@@ -338,6 +342,7 @@ export function changeServerType (serverType) {
 export function setServerParam (name, value) {
   return (dispatch, getState) => {
     dispatch({type: SET_SERVER_PARAM, serverType: getState().session.serverType, name, value});
+    getRunningSessions()(dispatch, getState);
```

Comment:
It does happen on every keystroke, I should debounce that.

And no we can't manually enter a session ID, I'll update that as well.